### PR TITLE
Make the time dependence in update more explicit.

### PR DESCRIPTION
**Pull request by davidsiska-vega**

### CODE REVIEW

**Review by qustavo — APPROVED** [review]

File: specs/0042-setting-fees-and-rewarding-lps.md [hunk 1]
```diff
@@ -93,33 +93,35 @@ The guiding principle of this section is that by committing stake a liquidity pr
 
 At any time let's say we have `market_value_proxy` calculated above and existing liquidity providers as below
 ```
-[MM 1 stake, MM 1 avg_entry_valuation]
-[MM 1 stake, MM 2 avg_entry_valuation]
+[LP 1 stake, LP 1 avg_entry_valuation]
```
Comment (qustavo) [inline specs/0042-setting-fees-and-rewarding-lps.md:96]: what is `1,1...N` in  `LP N stake`?

Comment (qustavo) [reply specs/0042-setting-fees-and-rewarding-lps.md:96]: same for `LP 1 avg_entry_valuation`

Comment (davidsiska-vega) [reply specs/0042-setting-fees-and-rewarding-lps.md:96]: We have `N` LPs in the market. (if an additional joins they'd be N+1).

Comment (davidsiska-vega) [reply specs/0042-setting-fees-and-rewarding-lps.md:96]: LP is Liquidity Provider (sorry we want to move from saying Market Makers to Liquidity Providers so I already changed here but perhaps that's causing confusion.

Comment (qustavo) [reply specs/0042-setting-fees-and-rewarding-lps.md:96]: I don't understand why we have:
```
[MM 1 stake, MM 1 avg_entry_valuation]
[MM 1 stake, MM 2 avg_entry_valuation]
```

were both have N = 1

Comment (davidsiska-vega) [reply specs/0042-setting-fees-and-rewarding-lps.md:96]: Ok, I see. That was a typo. I've fixed that.

Comment (qustavo) [reply specs/0042-setting-fees-and-rewarding-lps.md:96]: cool, thanks!

**Review by qustavo — APPROVED** [review]

**Pull request merged**